src/pages/experiments/[experimentId]/plots-and-tables/dot-plot/index.jsx

```diff
@@ -190,6 +192,8 @@ const DotPlotPage = (props) => {
   };
 
   const deleteData = (genes) => {
+    if (!plotData) return;
```

Comment:
Will this even be executed if theres no plotData - nothing to delete right? So maybe this is redundant?

Reply:
This executes when returning to the dot plot when there is an error loading the plot data, due to for example the work request timing out